docs/integrations/test-playbooks.md

```diff
@@ -99,27 +101,6 @@ version: -1
 name: IPInfo-Test
 ```
 
-## Adding Tests to conf.json
-In order to associate integrations with a test plabyook we mange a `conf.json` file (at the root of the repository). The conf.json file is located in the **Tests** directory.
```

Comment:
i wouldnt remove it, as in some cases it is still needed

instead, i would write that by default it is not required to add to `conf.json`, but if needed then use as follows (and keep the rest)

Reply:
If I add this back then definitely I would add the top 👆 note.